suites/pacific/upgrades/tier-2_scaleup_scaledown_after_upgrade_4x-to-5x-container.yaml

```diff
@@ -0,0 +1,598 @@
+#############################################################################
+# - Automation support for containerised upgrade from RHCS4 to RHCS5 cluster
+# - and test scale up and scale down of the cluster post upgrade
+# Cluster configuration: (conf/pacific/upgrades/tier-2_10node_cephadm_upgrade_scale.yaml)
```

Comment:
If the configuration file is modified, please change the filename here.

Reply:
Done.